Mathlib/Combinatorics/SimpleGraph/AdjMatrix.lean

```diff
@@ -44,9 +44,9 @@ variable {V α : Type*}
 namespace Matrix
 
 /-- `A : Matrix V V α` is qualified as an "adjacency matrix" if
-    (1) every entry of `A` is `0` or `1`,
-    (2) `A` is symmetric,
-    (3) every diagonal entry of `A` is `0`. -/
+(1) every entry of `A` is `0` or `1`,
+(2) `A` is symmetric,
+(3) every diagonal entry of `A` is `0`. -/
```

Comment:
This one probably should be a list, but that's a problem for a future PR.

Reply:
Agreed!